### PR TITLE
Fix Error View responseJSON Condition Checking

### PR DESCRIPTION
**Pull request by thayashi**

### CODE REVIEW

File: app/js/views/errorView.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ export default class ErrorView extends View {
     }
 
     this.message = response.statusText;
-    if (response.hasOwnProperty('responseJSON') && response.responseJSON !== (null || undefined)) {
+    if (response.hasOwnProperty('responseJSON') && response.responseJSON !== (null && undefined)) {
```
Comment (thealmightyone) [inline app/js/views/errorView.js]: This doesnt check if response.responseJSON is undefined.
try: 
`if (response.hasOwnProperty('responseJSON') && response.responseJSON)`

Comment (thayashi) [inline app/js/views/errorView.js]: you're right, let me fix

**Comment by thealmightyone** [comment]
I cna merge after squashing it :)

**Comment by stepek** [comment]
LGTM

**Pull request merged**